WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -28,7 +28,11 @@
       <fieldset <%= disable_form && "disabled" %>>
 
         <% if @competition.persisted? %>
-          <%= f.input :id %>
+          <<% if @competition_admin_view %>
+            <%= f.input :id %>
+          <% else %>
+            <%= f.input :id, disabled: true %>
```

Comment:
Since we want to disable the field if it's not in the admin view, it's probably cleaner to just use a one-liner:
```ruby
<%= f.input :id, disabled: !@competition_admin_view %>
```

Reply:
After Jacob's comment, then this line should rather be
```ruby
<%= f.input :id, disabled: is_actually_confirmed && !@competition_admin_view %>
```